README.md

```diff
@@ -106,3 +139,4 @@ AMaCC is based on the infrastructure of [c4](https://github.com/rswier/c4).
 * [Curated list of awesome resources on Compilers, Interpreters and Runtimes](http://aalhour.com/awesome-compilers/)
 * [Hacker News discussions](https://news.ycombinator.com/item?id=11411124)
 * [A Compiler Writing Journey](https://github.com/DoctorWkt/acwj) by Warren Toomey.
+* [How to solve "error while loading shared libraries" when trying to run an arm binary with qemu-arm?](https://stackoverflow.com/questions/16158994/how-to-solve-error-while-loading-shared-libraries-when-trying-to-run-an-arm-bi)
```

Comment:
Don't add this since we can improve the build system instead.

Reply:
Removed in latest commit.